css/index.css

```diff
@@ -0,0 +1,359 @@
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+* {
+  box-sizing: border-box;
+}
+html {
+  font-size: 62.5%;
+}
+html,
+body {
+  font-family: 'Ubuntu', sans-serif;
+  color: #5E6164;
+}
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Russo One', sans-serif;
+  color: #23293B;
+}
+h2 {
+  font-size: 3.2rem;
+}
+h3 {
+  font-size: 2.8rem;
+}
+p {
+  font-size: 1.6rem;
+  line-height: 1.4;
+}
+.container {
+  max-width: 800px;
+  width: 100%;
+  margin: 0 auto;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #F6F6F6;
+  padding-top: 2%;
+  margin-left: 18%;
+  margin-right: 18%;
+  padding-bottom: 1%;
+}
+@media (max-width: 500px) {
+  header {
+    max-width: 500px;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    width: 101%;
+  }
+}
+header h1 {
+  font-size: xx-large;
+  color: #454A59;
+}
+header .navbar {
+  position: fixed;
+  top: 0;
+  background: none;
+  width: 41%;
+  color: #23293B;
+  margin-top: 1%;
+  width: 40%;
+  display: flex;
+  flex-direction: row;
+  justify-items: center;
+  justify-content: space-between;
+}
+@media (max-width: 500px) {
+  header .navbar {
+    width: 90%;
+    margin-left: 8%;
+  }
+}
+header .navbar .navlink {
+  text-decoration: none;
+  color: #23293B;
+  width: 9%;
+}
+@media (max-width: 500px) {
+  header .navbar .navlink {
+    width: 20%;
+  }
+}
+header .navbar .navlink .navlink :hover {
+  color: white;
+}
+footer {
+  color: #22283B;
+  background: #f4f4f4;
+  padding: 20px 0;
+  border: 1px double #e0e0e0;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+footer p {
+  font-size: 0.9rem;
+}
+@media (max-width: 500px) {
+  footer {
+    font-size: xx-small;
+  }
+}
+@media (max-width: 500px) {
+  .home {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    max-width: 500px;
+  }
+}
+@media (max-width: 500px) {
+  .home .mainheader {
+    width: 500px;
+  }
+}
+.home .intro h2 {
+  padding: 15px;
+}
+.home .intro p {
+  padding: 0 15px;
+}
+.home .space-images {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+@media (max-width: 500px) {
+  .home .space-images {
+    justify-content: center;
+  }
```

Comment:
You can also add a width of 100% to this image media query and make it look more like the mobile ndesign file.